src/Sarif.Multitool.Library/Rules/SARIF2006.UrisShouldBeReachable.cs

```diff
@@ -36,7 +37,7 @@ public UrisShouldBeReachable()
         };
 
         // cache stores all visited Uris
-        private static readonly Dictionary<string, bool> s_checkedUris = new Dictionary<string, bool>();
+        private static readonly ConcurrentDictionary<string, bool> s_checkedUris = new ConcurrentDictionary<string, bool>();
```

Comment:
>ConcurrentDictionary [](http://example.com/codeflow?start=32&length=20)

when we run multiple tests, we are facing concurrency issues. To prevent that, i'm changing to ConcurrentDictionary